gethwrappers/abigen.go

```diff
@@ -0,0 +1,464 @@
+package gethwrappers
```

Comment:
where is this taken from?

Reply:
[here](https://github.com/smartcontractkit/chainlink/tree/281c853af4e63dffef5fa1ea0dd740c1a513177f/core/gethwrappers)